.github/workflows/unit.yml

```diff
@@ -0,0 +1,55 @@
+on: [push, pull_request]
+name: Unit test
+jobs:
+    tests:
+        name: PHP ${{ matrix.php }} / SF ^${{ matrix.symfony }}
+        runs-on: ubuntu-latest
+        strategy:
+            fail-fast: false
+            matrix:
+                php: [7.4, 8.0, 8.1]
+                symfony: [4.4.*, 5.4.*, 6.0.*]
+                exclude:
+                    - php: 7.4
+                      symfony: 6.0.*
+
+        services:
+            rabbitmq:
+                # 3.8.10 and later versions has enabled TLSv1.3 by default which is not supported by PHP images
+                # TLSv1.3 can be disabled by using configuration file, but not environment variables
```

Comment:
This comment looks weird to me. Is 3.8.10 disabling support for TLSv1.2 (supporting *only* 1.3) ? Or is the rabbitmq server or PHP shipping with a broken implementation of TLS negotiation ?

Reply:
You right, sorry it's a copy/paste error.

Reply:
copy-paste from where ?

Also, if you remove the comment explaining why 3.8.10 is not used without actually using new versions, this makes the situation even worse IMO.

Reply:
I just tryed with latest rabbitmq version 3.9 and it's working. 
Sorry I don't remember from where I pasted this part, I digged in so many repositories...